### PR TITLE
fix: do not try to validate input fields that are not present

### PR DESCRIPTION
**Pull request by cabelitos**
This commit fixes a problem where the ValidateDirectiveVisitor
would try to visit an input that is not present, thus caused
an undefined deference.

### CODE REVIEW

File: lib/ValidateDirectiveVisitor.test.ts [hunk 1]
```diff
@@ -1382,8 +1382,20 @@ directive @${name}(
...
+                field: String!
+                moreInputs: YetAnotherInput
+              }
+              input NullableSubInput {
+                field1: Args
+                field2: Args
+              }
               input DeepNullableInput {
                 nullable: Int @${name}
+                nullable2: NullableSubInput
```
Comment (barbieri) [inline lib/ValidateDirectiveVisitor.test.ts:1398]: I don't get why do you need these declarations to test, none are using the directive...
